documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc

```diff
@@ -11,23 +11,43 @@ If you want to remotely access {brandname} Server via JMX you must enable a remo
 ====
 endif::remote_caches[]
 
-.Procedure
+Enabling remote JMX involves exposing sensitive information about your system.
+You can enable remote JMX connections using one of the following approaches:
```

Comment:
I'm not sure about the first sentence tbh. It sounds like a cautionary note but doesn't quite blend with the following sentence. I would just drop it because we have a warning and other language that advises against the unsecured option.